### PR TITLE
feature/M2-518 calendar day view

### PR DESCRIPTION
**Pull request by IhorNesterenko99**

### CODE REVIEW

File: src/features/Applet/Schedule/Calendar/MonthView/MonthView.tsx [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 export const MonthView = (props: MonthViewType) => {
   const monthRef = useRef<MonthViewType>();
+  const { date, activeView } = props.components;
 
   useEffect(() => {
-    if (monthRef.current?.state?.rowLimit < 3 || monthRef.current?.state.needLimitMeasure) {
-      monthRef.current.setState({ rowLimit: 3 });
-    }
-  });
+    const timeout = setTimeout(() => {
```
Comment (wildkiomi) [inline src/features/Applet/Schedule/Calendar/MonthView/MonthView.tsx:13]: do you really need this timeout?

File: src/features/Applet/Schedule/Calendar/Calendar.styles.ts [hunk 2]
```diff
@@ -24,10 +25,117 @@ export const StyledAddBtn = styled(StyledClearedButton)`
...
+  .rbc-time-header-content {
+    padding: ${theme.spacing(1, 0.8)};
+    border: none;
+  }
+
+  .rbc-time-content {
+    padding-top: ${theme.spacing(1)};
+    border-top: ${variables.borderWidth.xl} solid ${variables.palette.surface_variant};
+
+    & > * + * > * {
```
Comment (kdanilenka32) [inline src/features/Applet/Schedule/Calendar/Calendar.styles.ts:57]: could y rewrite pls this selector

File: src/features/Applet/Schedule/Calendar/Calendar.styles.ts [hunk 3]
```diff
@@ -24,10 +25,117 @@ export const StyledAddBtn = styled(StyledClearedButton)`
...
+        color: ${variables.palette.outline};
+        font-size: ${variables.font.size.md};
+        line-height: ${variables.lineHeight.md};
+        letter-spacing: ${variables.letterSpacing.sm};
+      }
+    }
+  }
+
+  .rbc-time-header {
+    margin-right: 0 !important;
```
Comment (kdanilenka32) [inline src/features/Applet/Schedule/Calendar/Calendar.styles.ts:85]: no way without important?

File: src/features/Applet/Schedule/Calendar/Event/Event.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+export const Event = ({ title, event, uiType = UiType.Primary }: EventProps) => {
+  const { scheduledColor, startFlowIcon, start, end, allDayEvent, endAlertIcon, alwaysAvailable } =
+    event;
+  const isAllDayEvent = allDayEvent || alwaysAvailable;
+
+  return (
+    <StyledEvent>
+      <StyledTopWrapper>
+        <StyledLeftSection>
+          {uiType === UiType.Primary && scheduledColor && (
```
Comment (kdanilenka32) [inline src/features/Applet/Schedule/Calendar/Event/Event.tsx:32]: would be good move to var uiType === UiType.Primary and uiType === UiType.Secondary

File: src/features/Applet/Schedule/Calendar/Calendar.styles.ts (hunk 3 above)
Comment (IhorNesterenko99) [reply src/features/Applet/Schedule/Calendar/Calendar.styles.ts:85]: it's overwriting of the inline style

File: src/features/Applet/Schedule/Calendar/Event/Event.tsx (hunk 4 above)
Comment (IhorNesterenko99) [reply src/features/Applet/Schedule/Calendar/Event/Event.tsx:32]: done

File: src/features/Applet/Schedule/Calendar/Calendar.styles.ts (hunk 2 above)
Comment (IhorNesterenko99) [reply src/features/Applet/Schedule/Calendar/Calendar.styles.ts:57]: done

File: src/features/Applet/Schedule/Calendar/MonthView/MonthView.tsx (hunk 1 above)
Comment (IhorNesterenko99) [reply src/features/Applet/Schedule/Calendar/MonthView/MonthView.tsx:13]: yep, unfortunately without timeout I have no access to container elements, the are undefined

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**